core/i18n/index.ts

```diff
@@ -19,27 +19,20 @@ function setI18nLanguage (lang: string): string {
 }
 
 export function loadLanguageAsync (lang: string): Promise<string> {
-  if (!config.i18n.bundleAllStoreviewLanguages) {
```

Comment:
Wht this option has been removed? It was a solution for "lang blinking" and to not preload all the translations all the time